hr_citizenship/models/hr_employee.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import fields, models, api
+
+
+class HrEmployee(models.Model):
+    _inherit = "hr.employee"
+
+    citizenship_ids = fields.One2many(
+        comodel_name="res.citizenship",
+        inverse_name="employee_id",
+        string="Multi citizenship",
+    )
+
+    # override fields to update them with new model
+    identification_id = fields.Char(
+        compute="_compute_passport_data",
+        store="True",
```

Comment:
add an inverse or read only

Reply:
compute is readonly already

Reply:
ah confused it with related :man_facepalming: